.github/workflows/test_jax_dockerfile.yml

```diff
@@ -45,7 +43,7 @@ permissions:
 
 jobs:
   test_wheels:
-    name: Test | ${{ inputs.amdgpu_family }}
+    name: Test
```

Comment:
There is no `inputs.amdgpu_family` in this file [yet]

https://github.com/ROCm/TheRock/pull/2247 may delete this file though (cc @kiran-thumma )